components/admin_console/admin_definition.jsx

```diff
@@ -4393,7 +4393,7 @@ const AdminDefinition = {
                         label: t('admin.experimental.defaultTheme.title'),
                         label_default: 'Default Theme:',
                         help_text: t('admin.experimental.defaultTheme.desc'),
-                        help_text_default: 'Set a default theme that applies to all new users on the system.',
+                        help_text_default: 'Set a default theme that applies to all new users on the system with options "default", "organization", "mattermostDark", and "windows10".',
```

Comment:
Presumably you can enter any theme JSON here and it will become default. Otherwise we would make this a dropdown. Maybe mention that?

Reply:
@thefactremains ^^

Reply:
@crspeller Actually to be honest, I think it should be a drop down. The reason for this is because if you look underneath it, we pass these `Options`
![image](https://user-images.githubusercontent.com/17804942/88186524-47cff280-cc03-11ea-9953-554c3d795e7a.png)
However, when we build the component, they aren't getting used
![image](https://user-images.githubusercontent.com/17804942/88186563-54544b00-cc03-11ea-945a-942b605a29d9.png)

but if we look at the build drop down, options does get used
![image](https://user-images.githubusercontent.com/17804942/88186677-82398f80-cc03-11ea-8da6-42c6b2f1b920.png)

So gut feeling tells me that this should be a dropdown but I could be totally off here. I'm not sure it will actually accept any theme JSON.

Reply:
@crspeller Spoke with Dennis and we also agreed a dropdown would be better as I discussed with you.